addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzDiscoveryService.java

```diff
@@ -79,12 +80,27 @@ public AVMFritzDiscoveryService(BoxHandler bridgeHandler) {
         this.activate(null);
     }
 
+    @Override
+    protected void activate(Map<String, Object> configProperties) {
+        super.activate(configProperties);
+    }
+
+    @Override
+    protected void modified(Map<String, Object> configProperties) {
+        super.modified(configProperties);
+    }
+
+    public void deactivate() {
+        super.deactivate();
+        removeOlderResults(new Date().getTime());
```

Comment:
Why should valid results be removed here?

Reply:
In which situations will this service be deactivated? I can think of two: First the user deactivates it in his environment for a reason and second the binding gets unistalled. Are there more? And why should we leave them there?

Reply:
Well, if the service is shut down, if some parts are updated and bundle restart, etc.
The Inbox persists its entries, as long as they are valid, they simply should be kept as they are.

Reply:
You convinced me. Should I remove it in the `stopScan()` method too? ([AVMFritzDiscoveryService.java#L105](https://github.com/cweitkamp/openhab2-addons/blob/ad8b3ea1808903f1ac890335c6507cc2da081937/addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/discovery/AVMFritzDiscoveryService.java#L105))

Reply:
yes, I would think so.

Reply:
done